Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -37,6 +37,27 @@ fileprivate extension Range {
   }
 }
 
+/// Explicitly blacklisted `DocumentURI.pseudoPath` prefixes. No diagnostics will be emitted
+/// for these documents.
+fileprivate let documentPathPrefixBlacklist = [
```

Comment:
Style: Non-local variables should always have a type annotation.

Reply:
Done